src/backend/allocator/swapchain.rs

```diff
@@ -107,6 +109,26 @@ impl<B: Buffer> Deref for Slot<B> {
     }
 }
 
+impl<B: Buffer> Clone for Slot<B> {
+    fn clone(&self) -> Self {
+        Self(self.0.clone())
+    }
+}
```

Comment:
Hmm, no allowing users to clone the `Slot` was a very deliberate design decision, as most of the `Slot`s magic happens on drop. I wanted to avoid issues where users were unintentionally kept more references around and thus missed freeing the slot at the right time.

Does this need to be public (I don't see any usage of it in anvil, if I am not mistaken)? Or could it be an impl-function with proper docs to convey the use-case of this?

Reply:
This is a really good point. In fact this change would break the contract, Drop will always set acquired to false for each clone. I will revert that change as it is no longer necessary, if I am not mistaken I only added it for the frame experiment I already removed again.

Reply:
done